### DISCUSSION TITLE
Semantic Versioning

### DISCUSSION THREAD

**Discussion by ayg-qliktech** [aws/aws-sdk-net#2494] [General]
Hi. Our team is wondering why AWSSDK.S3 component doesn't follow semantic versioning (https://semver.org/) when releasing? One limitation we notice is our Dependabot PRs will not auto-merge since it expects semantic versions to do so (see below screenshot). Perhaps, this a bug with Dependabot, but there are many other benefits of semantic versioning outside of this. Are there any internal discussions or plans to move to semantic versioning? Thank you for your time and feedback in advance. 
<img width="1265" alt="image" src="https://user-images.githubusercontent.com/18533539/205362667-38a5049a-114f-4e95-90a2-66137909aefc.png">

**Comment by normj** [comment]
In the [README](https://github.com/aws/aws-sdk-net#versioning) is where we explaining our versioning. The main reason we don't follow semver exactly is we need to also express the relationship/compatibility between the individual service packages like S3 and the core package. So if you are using a 3.7.X version S3 you have to also use a 3.7.X of Core. It is very important when you are using multiple services because you want to be sure all service and packages and core need to be using versions in the same range. Making version bumps is a really big deal for AWS SDKs because it kicks in AWS deprecation policy and we don't want to take on that operational burden unless it is truly a major version change from the user's point of view.

**Reply by ayg-qliktech** [reply]
@normj: Thank you for the detailed explanation and apologies I missed it in the README.
